content/en/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md

```diff
@@ -2,7 +2,7 @@
 title: Upgrading Windows nodes
 min-kubernetes-server-version: 1.17
 content_type: task
-weight: 50
+weight: 100
```

Comment:
```suggestion
weight: 110
```
avoid using the same weight for 2 pages in a section, as this causes problems for localization

Reply:
Other than that, LGTM for SIG Docs

Reply:
> avoid using the same weight for 2 pages in a section, as this causes problems for localization

ok, I will update it.

Reply:
Addressed the above comment, PTAL!